tenma/tenmaDcLib.py

```diff
@@ -1048,3 +1051,209 @@ class Tenma72_13330(Tenma72_13320):
     MAX_MV = 30000
     #:
     SERIAL_EOL = "\n"
+
+class Tenma72_13360(Tenma72_13320):
+    MATCH_STR = ["72-13360"]
+    NCHANNELS = None
+    NCONFS = 5
+    MAX_MA = 15000
+    MAX_MV = 60000
+    SERIAL_EOL = "\n"
+    SERIAL_SETTER_SEPARATOR = ":"
+
+    def getStatus(self):
+        """
+            Returns the power supply status as a dictionary of values
+
+            "ch1Mode ": "C.V" or "C.C",
+            "output ": "ON" or "OFF",
+            "V/C priority ": "Current priority" or "Voltage priority",
+            "beep ": "ON" or "OFF",
+            "lock ": "ON" or "OFF",
+
+            :return: Dictionary of status values
+        """
+        self._sendCommand("STATUS?")
+        statusBytes = self._readBytes()
+
+        # 72-13360 sends two bytes back, the second being '\n'
+        status = statusBytes[0]
+
+        ch1mode = (status & 0b00000001)
+        output = (status & 0b00000010)
+        current_priority = (status & 0b00000100)
+        beep = (status & 0b00010000)
+        lock = (status & 0b00100000)
+
+        return {
+            "ch1Mode ": "C.V" if ch1mode else "C.C",
+            "output ": "ON" if output else "OFF",
+            "V/C priority ": "Current priority" if current_priority else "Voltage priority",
+            "beep ": "ON" if beep else "OFF",
+            "lock ": "ON" if lock else "OFF",
+        }
+    
+    def readCurrent(self, channel=""):
```

Comment:
I see, we will be able to skip the `-C` in `tenma-control` accept the default of 1 and just skip it by calling super with our `""` empty string